integrations/astra/src/haystack_integrations/components/retrievers/astra/retriever.py

```diff
@@ -33,13 +46,13 @@ def __init__(self, document_store: AstraDocumentStore, filters: Optional[Dict[st
 
     @component.output_types(documents=List[Document])
     def run(self, query_embedding: List[float], filters: Optional[Dict[str, Any]] = None, top_k: Optional[int] = None):
-        """Run the retriever on the given list of queries.
+        """Retrieve documents from the AstraDocumentStore.
 
-        Args:
-            query_embedding (List[str]): An input list of queries
-            filters (Optional[Dict[str, Any]], optional): A dictionary with filters to narrow down the search space.
-                Defaults to None.
-            top_k (Optional[int], optional): The maximum number of documents to retrieve. Defaults to None.
+        :param query_embedding: Floats representing the query embedding
```

Comment:
```suggestion
        :param query_embedding: floats representing the query embedding
```

Reply:
@davidsbatista Happy to adapt - here: https://github.com/deepset-ai/haystack/issues/7074 the names start upper case though. Which one is right? 🙈

Reply:
oh, good question - I don't know, I see more lower case than upper case, specially if you have multiple params